src/main/java/vazkii/botania/common/entity/EntityFallingStar.java

```diff
@@ -78,6 +91,10 @@ public void tick() {
 	protected void onHitEntity(@Nonnull EntityHitResult hit) {
 		super.onHitEntity(hit);
 		Entity e = hit.getEntity();
+		// Blacklisting villagers since trading with them counts as a "swing" and will summon a star.
```

Comment:
Trading with a villager counts as left clicking them?

Reply:
Oddly enough yes. So is opening a chest or a wooden door.